<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>670</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>23</x>
      <y>63</y>
      <width>724</width>
      <height>509</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_4">
     <item row="0" column="0">
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="label_choose_items">
         <property name="text">
          <string>Choose items here:</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_3">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>258</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_refresh_all">
         <property name="text">
          <string>Refresh All</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="1" column="0">
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QCheckBox" name="checkBox_mem_usage">
         <property name="text">
          <string>Memory Usage</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>268</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <layout class="QGridLayout" name="gridLayout">
         <item row="0" column="0">
          <widget class="QLabel" name="label_mem_total">
           <property name="text">
            <string>Mem Total (kB)</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLCDNumber" name="lcdNumber_mem_total">
           <property name="frameShadow">
            <enum>QFrame::Plain</enum>
           </property>
           <property name="digitCount">
            <number>10</number>
           </property>
           <property name="segmentStyle">
            <enum>QLCDNumber::Flat</enum>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="label_mem_free">
           <property name="text">
            <string>Mem Free (kB)</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QLabel" name="label_mem_avail">
           <property name="text">
            <string>Mem Avail (kB)</string>
           </property>
          </widget>
         </item>
         <item row="3" column="0">
          <widget class="QLCDNumber" name="lcdNumber_mem_free">
           <property name="frameShadow">
            <enum>QFrame::Plain</enum>
           </property>
           <property name="digitCount">
            <number>10</number>
           </property>
           <property name="segmentStyle">
            <enum>QLCDNumber::Flat</enum>
           </property>
          </widget>
         </item>
         <item row="3" column="1">
          <widget class="QLCDNumber" name="lcdNumber_mem_avail">
           <property name="frameShadow">
            <enum>QFrame::Plain</enum>
           </property>
           <property name="digitCount">
            <number>10</number>
           </property>
           <property name="segmentStyle">
            <enum>QLCDNumber::Flat</enum>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_mem_plot">
         <property name="text">
          <string>Plot</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="2" column="0">
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QCheckBox" name="checkBox_net_stats">
         <property name="text">
          <string>Network Stats</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>148</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <layout class="QGridLayout" name="gridLayout_2">
         <item row="0" column="0">
          <widget class="QLabel" name="label_net_lo">
           <property name="text">
            <string>Lo:</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QLabel" name="label_net_receive_lo">
           <property name="text">
            <string>Rx (Bytes)</string>
           </property>
          </widget>
         </item>
         <item row="0" column="2">
          <widget class="QLabel" name="label_net_transmit_lo">
           <property name="text">
            <string>Tx (Bytes)</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QLCDNumber" name="lcdNumber_net_receive_lo">
           <property name="frameShadow">
            <enum>QFrame::Plain</enum>
           </property>
           <property name="digitCount">
            <number>10</number>
           </property>
           <property name="segmentStyle">
            <enum>QLCDNumber::Flat</enum>
           </property>
          </widget>
         </item>
         <item row="1" column="2">
          <widget class="QLCDNumber" name="lcdNumber_net_transmit_lo">
           <property name="frameShadow">
            <enum>QFrame::Plain</enum>
           </property>
           <property name="digitCount">
            <number>10</number>
           </property>
           <property name="segmentStyle">
            <enum>QLCDNumber::Flat</enum>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="label_net_eth">
           <property name="text">
            <string>Eth:</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QLabel" name="label_net_receive_eth">
           <property name="text">
            <string>Rx (Bytes)</string>
           </property>
          </widget>
         </item>
         <item row="2" column="2">
          <widget class="QLabel" name="label_net_transmit_eth">
           <property name="text">
            <string>Tx (Bytes)</string>
           </property>
          </widget>
         </item>
         <item row="3" column="1">
          <widget class="QLCDNumber" name="lcdNumber_net_receive_eth">
           <property name="frameShadow">
            <enum>QFrame::Plain</enum>
           </property>
           <property name="digitCount">
            <number>10</number>
           </property>
           <property name="segmentStyle">
            <enum>QLCDNumber::Flat</enum>
           </property>
          </widget>
         </item>
         <item row="3" column="2">
          <widget class="QLCDNumber" name="lcdNumber_net_transmit_eth">
           <property name="frameShadow">
            <enum>QFrame::Plain</enum>
           </property>
           <property name="digitCount">
            <number>10</number>
           </property>
           <property name="segmentStyle">
            <enum>QLCDNumber::Flat</enum>
           </property>
          </widget>
         </item>
         <item row="4" column="0">
          <widget class="QLabel" name="label_net_eth_3">
           <property name="text">
            <string>Wl:</string>
           </property>
          </widget>
         </item>
         <item row="4" column="1">
          <widget class="QLabel" name="label_net_receive_wl">
           <property name="text">
            <string>Rx (Bytes)</string>
           </property>
          </widget>
         </item>
         <item row="4" column="2">
          <widget class="QLabel" name="label_net_transmit_wl">
           <property name="text">
            <string>Tx (Bytes)</string>
           </property>
          </widget>
         </item>
         <item row="5" column="1">
          <widget class="QLCDNumber" name="lcdNumber_net_receive_wl">
           <property name="frameShadow">
            <enum>QFrame::Plain</enum>
           </property>
           <property name="digitCount">
            <number>10</number>
           </property>
           <property name="segmentStyle">
            <enum>QLCDNumber::Flat</enum>
           </property>
          </widget>
         </item>
         <item row="5" column="2">
          <widget class="QLCDNumber" name="lcdNumber_net_transmit_wl">
           <property name="frameShadow">
            <enum>QFrame::Plain</enum>
           </property>
           <property name="digitCount">
            <number>10</number>
           </property>
           <property name="segmentStyle">
            <enum>QLCDNumber::Flat</enum>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_net_plot">
         <property name="text">
          <string>Plot</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="3" column="0">
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <widget class="QCheckBox" name="checkBox_cpu_temp">
         <property name="text">
          <string>CPU Temperatures</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_4">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>228</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <layout class="QGridLayout" name="gridLayout_3">
         <item row="0" column="0">
          <widget class="QLabel" name="label_cpu_temp_zone0">
           <property name="text">
            <string>Zone 0 (deg C)</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QLabel" name="label_cpu_temp_zone1">
           <property name="text">
            <string>Zone 1 (deg C)</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLCDNumber" name="lcdNumber_cpu_temp_zone0">
           <property name="frameShadow">
            <enum>QFrame::Plain</enum>
           </property>
           <property name="digitCount">
            <number>10</number>
           </property>
           <property name="segmentStyle">
            <enum>QLCDNumber::Flat</enum>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QLCDNumber" name="lcdNumber_cpu_temp_zone1">
           <property name="frameShadow">
            <enum>QFrame::Plain</enum>
           </property>
           <property name="digitCount">
            <number>10</number>
           </property>
           <property name="segmentStyle">
            <enum>QLCDNumber::Flat</enum>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="label_cpu_temp_zone2">
           <property name="text">
            <string>Zone 2 (deg C)</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QLabel" name="label_cpu_temp_zone3">
           <property name="text">
            <string>Zone 3 (deg C)</string>
           </property>
          </widget>
         </item>
         <item row="3" column="0">
          <widget class="QLCDNumber" name="lcdNumber_cpu_temp_zone2">
           <property name="frameShadow">
            <enum>QFrame::Plain</enum>
           </property>
           <property name="digitCount">
            <number>10</number>
           </property>
           <property name="segmentStyle">
            <enum>QLCDNumber::Flat</enum>
           </property>
          </widget>
         </item>
         <item row="3" column="1">
          <widget class="QLCDNumber" name="lcdNumber_cpu_temp_zone3">
           <property name="frameShadow">
            <enum>QFrame::Plain</enum>
           </property>
           <property name="digitCount">
            <number>10</number>
           </property>
           <property name="segmentStyle">
            <enum>QLCDNumber::Flat</enum>
           </property>
          </widget>
         </item>
         <item row="4" column="0">
          <widget class="QLabel" name="label_cpu_temp_zone4">
           <property name="text">
            <string>Zone 4 (deg C)</string>
           </property>
          </widget>
         </item>
         <item row="4" column="1">
          <widget class="QLabel" name="label_cpu_temp_zone5">
           <property name="text">
            <string>Zone 5 (deg C)</string>
           </property>
          </widget>
         </item>
         <item row="5" column="0">
          <widget class="QLCDNumber" name="lcdNumber_cpu_temp_zone4">
           <property name="frameShadow">
            <enum>QFrame::Plain</enum>
           </property>
           <property name="digitCount">
            <number>10</number>
           </property>
           <property name="segmentStyle">
            <enum>QLCDNumber::Flat</enum>
           </property>
          </widget>
         </item>
         <item row="5" column="1">
          <widget class="QLCDNumber" name="lcdNumber_cpu_temp_zone5">
           <property name="frameShadow">
            <enum>QFrame::Plain</enum>
           </property>
           <property name="digitCount">
            <number>10</number>
           </property>
           <property name="segmentStyle">
            <enum>QLCDNumber::Flat</enum>
           </property>
          </widget>
         </item>
         <item row="6" column="0">
          <widget class="QLabel" name="label_cpu_temp_zone6">
           <property name="text">
            <string>Zone 6 (deg C)</string>
           </property>
          </widget>
         </item>
         <item row="6" column="1">
          <widget class="QLabel" name="label_cpu_temp_zone7">
           <property name="text">
            <string>Zone 7 (deg C)</string>
           </property>
          </widget>
         </item>
         <item row="7" column="0">
          <widget class="QLCDNumber" name="lcdNumber_cpu_temp_zone6">
           <property name="frameShadow">
            <enum>QFrame::Plain</enum>
           </property>
           <property name="digitCount">
            <number>10</number>
           </property>
           <property name="segmentStyle">
            <enum>QLCDNumber::Flat</enum>
           </property>
          </widget>
         </item>
         <item row="7" column="1">
          <widget class="QLCDNumber" name="lcdNumber_cpu_temp_zone7">
           <property name="frameShadow">
            <enum>QFrame::Plain</enum>
           </property>
           <property name="digitCount">
            <number>10</number>
           </property>
           <property name="segmentStyle">
            <enum>QLCDNumber::Flat</enum>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_cpu_temp_plot">
         <property name="text">
          <string>Plot</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <widget class="QMenu" name="menuAbout">
    <property name="title">
     <string>About</string>
    </property>
    <addaction name="actionInfo"/>
   </widget>
   <addaction name="menuAbout"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionInfo">
   <property name="icon">
    <iconset theme="help-about">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>Info</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
